Exiled.API/Features/Intercom.cs

```diff
@@ -0,0 +1,59 @@
+// -----------------------------------------------------------------------
+// <copyright file="Intercom.cs" company="Exiled Team">
+// Copyright (c) Exiled Team. All rights reserved.
+// Licensed under the CC BY-SA 3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Exiled.API.Features
+{
+    /// <summary>
+    /// A set of tools to easily handle the Intercom.
+    /// </summary>
+    public static class Intercom
+    {
+        /// <summary>
+        /// Gets or sets the text displayed on the intercom screen.
+        /// </summary>
+        public static string DisplayText
+        {
+            get => global::Intercom.host.CustomContent;
+            set => global::Intercom.host.CustomContent = value;
+        }
+
+        /// <summary>
+        /// Gets or sets the current state of the intercom.
+        /// </summary>
+        public static global::Intercom.State State
+        {
+            get => global::Intercom.host.IntercomState;
+            set => global::Intercom.host.IntercomState = value;
+        }
+
+        /// <summary>
+        /// Gets a value indicating whether or not the intercom is currently being used.
+        /// </summary>
+        public static bool InUse => State == global::Intercom.State.Transmitting || State == global::Intercom.State.TransmittingBypass || State == global::Intercom.State.AdminSpeaking;
```

Comment:
Couldnt you just check if its in cooldown or waiting

Reply:
> Couldnt you just check if its in cooldown or waiting

What's the purpose of that? I could check ``Intercom.host.remainingCooldown``, however, that gets changed to ``3f`` when the player just pressed the button.

Reply:
The purpose of it is checking less states

Reply:
Instead of 3, 2